### DISCUSSION TITLE
Is there a simple way to give the player character a custom name and portrait?

### DISCUSSION THREAD

**Discussion by ghost** [dialogic-godot/dialogic#706] [Q&A] [answered]
Just wondering, since the player character in the game I'm working on has a customizable name and gender.

**Comment by Jowan-Spooner** [comment, marked as answer]
Alright I'm going to answer this, sorry I didn't see it ealier.

Customizing the name is pretty easy. You should create a value definition and call it something like PLAYER_NAME.
Then you can create a name selection/editing screen on your own (basically you only need a LineEdit and a Button). Set the definition to your name by using `Dialogic.set_variable('PLAYER_NAME', $LineEdit.text)`.
Now you can use `[PLAYER_NAME]` (with the brackets) inside your text, question or choice events and it will be replaced with the name.

The portraits aren't that easy right now. You could create a custom scene for your portraits that change the image depending on some variable. You can also set the portrait depending on a Value definition but that is pretty limited, so I'm not sure if that's any good.

Hope this helps anyone.
